src/main/tools/travis/deploy

```diff
@@ -0,0 +1,42 @@
+#! /bin/bash
+
+#
+# A simple script to publish our artifacts on a push to the "master" branch.
```

Comment:
branch name should be "main"

Reply:
Yes, though this is a bug report/change for the parent project not this project since this project just inherited this file. I'll add an issue to freelib-parent for this, thanks :-)